backend/src/main/java/com/cruru/advice/CruruCustomException.java

```diff
@@ -0,0 +1,14 @@
+package com.cruru.advice;
+
+import lombok.Getter;
+
+@Getter
+public class CruruCustomException extends RuntimeException {
+
+    private final String statusCode;
+
+    public CruruCustomException(String message, String statusCode) {
+        super(message);
+        this.statusCode = statusCode;
+    }
```

Comment:
필드가 String이 아닌 int가 가능하다면, HttpStatus의 value를 활용할 수 있을 것 같아요.

```suggestion
    private final int statusCode;

    public CruruCustomException(String message, HttpStatus status) {
        super(message);
        this.statusCode = status.value();
    }
```